{"name": "注册失败", "status": "passed", "description": "测试用户注册失败", "steps": [{"name": "执行测试用例: 手机号格式错误（包含字母）时注册失败", "status": "passed", "steps": [{"name": "填写注册信息", "status": "passed", "start": 1751191754234, "stop": 1751191754638}, {"name": "验证错误信息", "status": "passed", "start": 1751191754670, "stop": 1751191754678}], "start": 1751191754144, "stop": 1751191754678}], "parameters": [{"name": "test_case", "value": "{'id': 'invalid_phone_format_letters', 'email': 'invalid-phone2@example.com', 'password': 'password123', 'password_confirmation': 'password123', 'username': 'phone test user 2', 'rank': 'normal', 'address': 'San Antonio, TX', 'tel': 'abcdefghijk', 'gender': '9', 'birthday': '1989-06-18', 'notification': False, 'expected_messages': {'email': '', 'password': '', 'password_confirmation': '', 'username': '', 'address': '', 'tel': 'Please match the requested format.', 'gender': '', 'birthday': ''}, 'description': '手机号格式错误（包含字母）时注册失败'}"}], "start": 1751191754144, "stop": 1751191754679, "uuid": "e02e099b-0ed0-4a38-9c1d-06da745cd815", "historyId": "1197f8982cd20f543e9fb858a94a957f", "testCaseId": "5a59ec60b5050f5d2e8136bb5d5a68e3", "fullName": "testcase.test_signup.TestSignup#test_signup_failure", "labels": [{"name": "feature", "value": "注册功能"}, {"name": "story", "value": "注册失败"}, {"name": "tag", "value": "order(2)"}, {"name": "tag", "value": "@pytest.mark.usefixtures('driver')"}, {"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_signup"}, {"name": "subSuite", "value": "TestSignup"}, {"name": "host", "value": "SQG"}, {"name": "thread", "value": "19712-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_signup"}]}